Unable to run on macOS

Running `make run` yields the following: 

```
cargo:rustc-link-search=native=/Users/tublitzed/aaarach/Codebase/GIT/syncstorage-rs/target/debug/build/libz-sys-153843b33fd7f4b5/out/lib
running: "cmake" "/Users/tublitzed/.cargo/registry/src/github.com-1ecc6299db9ec823/grpcio-sys-0.6.0/grpc" "-DgRPC_INSTALL=false" "-DgRPC_BUILD_CSHARP_EXT=false" "-DgRPC_BUILD_CODEGEN=false" "-DgRPC_BENCHMARK_PROVIDER=none" "-DgRPC_SSL_PROVIDER=package" "-DgRPC_ZLIB_PROVIDER=package" "-DCMAKE_INSTALL_PREFIX=/Users/tublitzed/aaarach/Codebase/GIT/syncstorage-rs/target/debug/build/grpcio-sys-0b3e427acdd6680a/out" "-DCMAKE_C_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64 -arch x86_64" "-DCMAKE_C_COMPILER=/usr/bin/cc" "-DCMAKE_CXX_FLAGS= -stdlib=libc++ -ffunction-sections -fdata-sections -fPIC -m64 -arch x86_64" "-DCMAKE_CXX_COMPILER=/usr/bin/c++" "-DCMAKE_ASM_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64 -arch x86_64" "-DCMAKE_ASM_COMPILER=/usr/bin/cc" "-DCMAKE_BUILD_TYPE=Debug"
-- Configuring incomplete, errors occurred!
See also "/Users/tublitzed/aaarach/Codebase/GIT/syncstorage-rs/target/debug/build/grpcio-sys-0b3e427acdd6680a/out/build/CMakeFiles/CMakeOutput.log".
See also "/Users/tublitzed/aaarach/Codebase/GIT/syncstorage-rs/target/debug/build/grpcio-sys-0b3e427acdd6680a/out/build/CMakeFiles/CMakeError.log".

--- stderr
CMake Warning at cmake/protobuf.cmake:51 (message):
  gRPC_PROTOBUF_PROVIDER is "module" but PROTOBUF_ROOT_DIR is wrong
Call Stack (most recent call first):
  CMakeLists.txt:207 (include)


CMake Error at /usr/local/Cellar/cmake/3.18.1/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:165 (message):
  Could NOT find OpenSSL, try to set the path to OpenSSL root folder in the
  system variable OPENSSL_ROOT_DIR (missing: OPENSSL_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.18.1/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:458 (_FPHSA_FAILURE_MESSAGE)
  /usr/local/Cellar/cmake/3.18.1/share/cmake/Modules/FindOpenSSL.cmake:486 (find_package_handle_standard_args)
  cmake/ssl.cmake:66 (find_package)
  CMakeLists.txt:208 (include)
```

Looks like a revert on [this addition](https://github.com/mozilla-services/syncstorage-rs/commit/76b577b0d23ec9d752ed1b1ab05b18efb862077d#diff-80398c5faae3c069e4e6aa2ed11b28c0R71) should resolve, but that breaks things for ubuntu so we should find another way to address this.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -34,6 +34,9 @@ env_logger = "0.7.1"
 failure = "0.1.8"
 futures = { version = "0.3", features = ["compat"] }
 googleapis-raw = { version = "0", path = "vendor/mozilla-rust-sdk/googleapis-raw" }
+# Some versions of OpenSSL 1.1.1 conflict with grpcio's built-in boringssl which can cause
+# syncserver to either fail to either compile, or start. In those cases, try
+# `cargo build --features grpcio/openssl ...`
 grpcio = { version = "0.6.0" }
 lazy_static = "1.4.0"
 hawk = "3.2"
@@ -67,9 +70,6 @@ uuid = { version = "0.8.1", features = ["serde", "v4"] }
 validator = "0.10"
 validator_derive = "0.10"
 woothee = "0.11"
-
-[target.'cfg(target_vendor="ubuntu")'.dependencies]
-grpcio = { version="0.6", features=["openssl"] }
 
 [dev-dependencies]
 tokio = { version = "0.2", features = ["macros"] }

--- a/README.md
+++ b/README.md
@@ -122,7 +122,7 @@ This requires access to the mozilla-rust-sdk which is now available at `/vendor/
 
 ### Connecting to Firefox
 
-This will walk you through the steps to connect this project to your local copy of Firefox. 
+This will walk you through the steps to connect this project to your local copy of Firefox.
 
 1. Follow the steps outlined above for running this project using [MySQL](https://github.com/mozilla-services/syncstorage-rs#mysql).
 
@@ -201,10 +201,12 @@ Open a PR after doing the following:
 
 Once your PR merges, then go ahead and create an official [GitHub release](https://github.com/mozilla-services/syncstorage-rs/releases).
 
-
 ## Troubleshooting
 
 - `rm Cargo.lock; cargo clean;` - Try this if you're having problems compiling.
+
+- Some versions of OpenSSL 1.1.1 can conflict with grpcio's built in BoringSSL. These errors can cause syncstorage to fail to run or compile.
+If you see a problem related to `libssl` you may need to specify the `cargo` option `--features grpcio/openssl`  to force grpcio to use OpenSSL.
 
 ## Related Documentation
 

